db/migrate/20170207192912_encrypt_otp_secret.rb

```diff
@@ -0,0 +1,22 @@
+class EncryptOtpSecret < ActiveRecord::Migration
+  class User < ActiveRecord::Base
+  end
+
+  def up
+    encrypt_otp_secret_key
+  end
+
+  def down
+  end
+
+  def encrypt_otp_secret_key
+    User.where.not(otp_secret_key: nil).find_in_batches.with_index do |users, batch|
+      users.each do |user|
+        encrypted_attribute = EncryptedAttribute.new_from_decrypted(user.otp_secret_key)
```

Comment:
I agree with Peter that "we have a problem right now where code is living too long in master before being deployed everywhere." I also think Zach makes a very good point. This particular migration is only meant to be run on servers where we don't want to wipe out the DB AND where we already have TOTP-enabled users. If a server has a fresh DB, this migration does not need to be run, and the code will work just fine. Given that we've been wiping out the DB on most of our servers, except for demo, I think we only need to run this on demo as a one-off task.